src/plugins/intel_npu/tests/functional/behavior/npuw/behavior_tests.cpp

```diff
@@ -0,0 +1,261 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior_tests.hpp"
+#include "openvino/opsets/opset11.hpp"
+#include "openvino/runtime/exec_model_info.hpp"
+#include "openvino/runtime/internal_properties.hpp"
+#include "openvino/runtime/make_tensor.hpp"
+#include "openvino/runtime/properties.hpp"
+#include "openvino/util/file_util.hpp"
+#include "openvino/util/shared_object.hpp"
+#include "npuw_private_properties.hpp"
+
+using namespace testing;
+using namespace ov::npuw::tests;
+
+std::shared_ptr<ov::Model> BehaviorTestsNPUW::create_example_model() {
+    auto param = std::make_shared<ov::opset11::Parameter>(ov::element::i64, ov::PartialShape{1, 3, 2, 2});
+    param->set_friendly_name("input");
+    auto const_value = ov::opset11::Constant::create(ov::element::i64, ov::Shape{1, 1, 1, 1}, {1});
+    const_value->set_friendly_name("const_val");
```

Comment:
Just in case, why `i64` for this simple arith? Not the most obvious choice.

Reply:
It just was copied from some network implementations in tests..